resources/osl_mysql_dev.rb

```diff
@@ -0,0 +1,31 @@
+provides :osl_mysql_dev
+resource_name :osl_mysql_dev
+unified_mode true
+
+default_action :create
+
+property :database, String, name_property: true
+property :username, String, required: true
+property :password, String, required: true
+
+# Install the mariadb package, set up the service, set up the user, then set up the given database
+action :create do
+  # Install the package, and set up the service
+  mariadb_server_install 'MariaDB' do
+    password 'iloveinsecurepasswords'
```

Comment:
Let's create a new property called `server_password` and have it default to `osl_mysql_test`

Reply:
I've set up an extra property to manage the root password.